module.js

```diff
@@ -0,0 +1,78 @@
+var dateUtil = (function() {
+  var privateDate = null;
+
+  function initDate(){
+    privateDate = new Date();
+    return privateDate;
+  }
+
+  function setDate(date) {
+    if(typeof date === 'undefined')
```

Comment:
I strongly encourage you to use curly braces in your code for conditionals (`if` and `else` blocks.) JavaScript is not a whitespace formatted language, so this is a bad habit to get into with this language.